docs/index.html.ejs

```diff
@@ -712,14 +712,34 @@ import "7.css/dist/gui/tabs.css"</pre>
           specified will receive a horizontal line appended after itself.
         </p>
 
+        <p>
+          To accommodate an extra piece of info for a menu item (like a shortcut mnemonic)
+          on the right side, nest it with an <code>i</code> element.
```

Comment:
`<i>` is not the appropriate tag for this, especially when the `font-style` has been overridden below. Consider using `<span` which is used to supply markup for grouped elements, grouped due to there being multiple heterogenous characters, e.g. `F` and `2` for `F2`; `Ctrl`, `+`, `X` for `Ctrl + X`; so on and so forth.